.github/workflows/astria-build-and-publish-image.yml

```diff
@@ -5,13 +5,13 @@ on:
   workflow_dispatch:
   push:
     branches:
-      - "main"
+      - "release/geth-1.13.x"
     tags:
-      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+-geth-1.13.x"
```

Comment:
I personally would prefer to prefix the tag with geth-1.13.x and then suffix with the version. not a major thing

Reply:
As noted in seperate conversation, this is purposeful. I consider this more of a build identifier. The semantic version ahead should be generally compatible astria-geth versions. It's just a build detail of which geth version it is.

You can run `v0.11.0` and `v0.11.0-geth-1.13.x` images can be run and reach the same state against the same other astria software. If we but this information ahead it implies they are not compatible.